docs/usage/downloading-and-deploying.md

```diff
@@ -56,3 +56,40 @@ services.
+| pyodide.js        | Exports loadPyodide on globalThis. Legacy support for people who can't use es6 modules for whatever reason. Prefer pyodide.mjs.                                                                                                                   |
+| pyodides.mjs      | A small JS loader shim which exports loadPyodide. It manages downloading the runtime and handling user settings.                                                                                                                                  |
+| python_stdlin.zip | The Python Standard Library for pyodide. A zip file consisting of the Python Lib folder (except a few things we've unvendored) and the Pyodide Python runtime libraries. Mounted directly into the Pyodide FS and imported using ZipImporter      |
+| package.json      | Tells node how to use Pyodide, since pyodide-core was primarily intended for use with node.                                                                                                                                                       |
+| pyodide-lock.json | Lockfile for Python packages, used by pyodide.loadPackage and micropip.install. Necessary in all cases.                                                                                                                                           |
+
+### Full Release Additional Files
+
+| File Name               | Description                                                                                                                                                                                          |
+| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| fonts/                  | Used by matplotlib-pyodide.                                                                                                                                                                          |
+| ffi.d.ts                | Typescript type definitions, useful if you want to use Pyodide in a typescript project                                                                                                               |
+| pyodide.d.ts            | Typescript type definitions, useful if you want to use Pyodide in a typescript project                                                                                                               |
+| 227 \*.whl              | Contains various Python Wheels like NumPy, Pandas, SciPy, etc. When downloading and including in a project, these no longer need to be loaded from Pyodide's online wheel repository during runtime. |
```

Comment:
These numbers are unlikely to stay up to date... We might need a script to check this.

Reply:
I removed the number of files. It is kind of unnecessary to include the exact number tbf.